hack/make/run

```diff
@@ -33,12 +33,32 @@ if [ "$DOCKER_REMAP_ROOT" ]; then
 	extra_params="$extra_params --userns-remap $DOCKER_REMAP_ROOT"
 fi
 
+
+if [ -n "$DOCKER_EXPERIMENTAL" ]; then
+	extra_params="$extra_params --experimental"
+fi
+
+dockerd="dockerd"
+socket=/var/run/docker.sock
+if [ -n "$DOCKER_ROOTLESS" ]; then
+    if [ -z "$DOCKER_EXPERIMENTAL" ]; then
+        echo >&2 '# DOCKER_ROOTLESS requires DOCKER_EXPERIMENTAL to be set'
+        exit 1
+    fi
+    user="unprivilegeduser"
+    uid=$(id -u $user)
+    mkdir -p -m 700 /tmp/docker-${uid}
+    chown $user /tmp/docker-${uid}
+    dockerd="sudo -u $user -E XDG_RUNTIME_DIR=/tmp/docker-${uid} -E HOME=/home/${user} -- dockerd-rootless.sh"
+    socket=/tmp/docker-${uid}/docker.sock
+fi
+
 args="--debug \
-	--host tcp://0.0.0.0:${listen_port} --host unix:///var/run/docker.sock \
+	--host tcp://0.0.0.0:${listen_port} --host unix://${socket} \
```

Comment:
Don't know if needed, because this whole block is already inside quotes;

```suggestion
	--host "tcp://0.0.0.0:${listen_port}" --host "unix://${socket}"" \
```

Reply:
fixed